dist/gs-board.html

```diff
@@ -1 +1 @@
n this.table[e.rowIndex][e.cellIndex][t]=i,this.fire("board-changed")},fillTable:function(){var e,t,i,n,r,o,s,a,l;if(null!=this.table){for(i=function(e,t){return e.slice(0,t)},n=this.table.slice().reverse(),e=o=0,a=this.size.y;0<=a?o<a:o>a;e=0<=a?++o:--o)for(t=s=0,l=this.size.x;0<=l?s<l:s>l;t=0<=l?++s:--s)null==n[e]&&(n[e]=[]),n[e]=i(n[e],this.size.x),null==(r=n[e])[t]&&(r[t]={});return n=i(n,this.size.y),this.table=n.reverse(),this._forceHeaderSet()}},update:function(e,t){return this.table=e,this.header=t,this._setSizeFromTable(),this.fillTable()},detectAttire:function(){return this.attireSrc?this.attire=GobstonesBoard.getAttire(this.attireSrc):GobstonesBoard.defaultAttire&&!this.noAttire?this.attire=GobstonesBoard.defaultAttire:void 0},_initializeTable:function(){return null!=this.table?this._setSizeFromTable():(this.table=[],this.fillTable())},_setSizeFromTable:function(){var e;return this.size={x:(null!=(e=this.table[0])?e.length:void 0)||0,y:this.table.length||0}},_initializeOptions:function(){var e;return null==this.options&&(this.options={}),null!=(e=this.options).editable?e.editable:e.editable=!1},_updateSize:function(){var e;return this.columnIndexes=function(){e=[];for(var t=0,i=this.size.x;0<=i?t<i:t>i;0<=i?t++:t--)e.push(t);return e}.apply(this),this.fire("board-changed")},_forceHeaderSet:function(){var e,t;return e=Math.min(this.header.x,this.size.x-1),t=Math.min(this.header.y,this.size.y-1),this.header=null,this.header={x:e,y:t}},_readGbb:function(){var e,t,i,n;if(!(null==(e=Polymer.dom(this).textContent)||e.indexOf("GBB")<0))return this.gbb=e,n=gbbReader.fromString(this.gbb),this.table=n.table,this.header=n.head,i=n.width,t=n.height,this.size.x=i,this.size.y=t,this.detectAttire()},_updateStyles:function(e,t,i){var n,r,o;if(o=null!=e?e.base:void 0,r=null!=t?t.base:void 0,n=null!=i?i.base:void 0,null!=o&&null!=r)return null!=n&&n.enabled?this._setBorderOff():this._setBorderOn(),this._updateBorderImages(n),this.updateStyles(),this.querySelectorAll("gs-cell").forEach(function(e){return e.updateCellStyles(o,r,n)})},_setBorderOn:function(){return this.customStyle["--cell-padding"]="2px",this.customStyle["--cell-border"]="solid #888 1px"},_setBorderOff:function(){return this.customStyle["--cell-padding"]="0 0",this.customStyle["--cell-border"]="none"},_updateBorderImages:function(e){var t;return t=function(t){return function(t){var i,n;if(i=null!=e&&null!=(n=e.borders)?n[t]:void 0,(null!=e?e.enabled:void 0)&&null!=i)return"url("+i+")"}}(),this.customStyle["--top-left-background-url"]=t("topLeft"),this.customStyle["--top-right-background-url"]=t("topRight"),this.customStyle["--bottom-left-background-url"]=t("bottomLeft"),this.customStyle["--bottom-right-background-url"]=t("bottomRight"),this.customStyle["--left-background-url"]=t("left"),this.customStyle["--top-background-url"]=t("top"),this.customStyle["--right-background-url"]=t("right"),this.customStyle["--bottom-background-url"]=t("bottom")}});</script></dom-module></div></body></html>
```

Comment:
This generated file has a difference: `assetpath="dist/components/"`

Suspicious :thinking:. Is this ok? @rodri042